README.md

```diff
@@ -52,6 +52,13 @@ The easiest way to try this out is to run the standalone server via `brokeneck-f
 
 To easily develop the packages of this repo you can execute:
 
+- `cd packages/brokeneck-html`
+- `cp .env.sample .env`
+- configure `.env` with the correct REACT_APP_API_PATH. The sample one works on the default local install.
+- `cd ../../`
```

Comment:
shall we take it a step further and add a npm script in fastify and html packages which generates the .env files from the .env.sample files if they don't already exist? so we don't have to make the instructions so verbose. it would apply to the previous paragraph as well. It could look something like this:

```
"create:env": "node -r fs -e \"fs.copyFileSync('.env.sample', '.env', fs.constants.COPYFILE_EXCL)\""
```

and it would be in both packages. then the instruction would simply mention to run `lerna run create:env:`

Reply:
Yes, I thought about this as well.
That's a good idea. I'll update the scripts and docs 👍